google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -1256,7 +1262,7 @@ static final class AppendRequestAndResponse {
 
     TimedAttemptSettings attemptSettings;
 
-    Instant requestCreationTimeStamp;
+    Instant requestSendTimeStamp;
```

Comment:
Add some comment about this value, it is the timestamp where request hit the network.

Reply:
Done.